core-plugins/src/main/java/io/cdap/plugin/batch/action/EmailAction.java

```diff
@@ -193,6 +212,12 @@ public static class Config extends ConditionConfig {
     @Macro
     private Boolean includeWorkflowToken;
 
+    @Nullable
+    @Description("Optional property that can be used to pass specific set of javamail properties. The syntax " +
+            "is 'property name:property value'. If this property is not set, default values are chosen.")
```

Comment:
Are the properties specified in `<property_name>:<value>` format? The `keyvalue` widget used below seems to imply that it is delimited by `,` and uses `=` to differentiate key and value.